### PR TITLE
CIP-95, CIP-105, CPS-10 | Update CIPs/CPSs to reflect new optional sections + update email

### PR DESCRIPTION
**Pull request by Ryun1**
- Reflect new optional sections for acknowledgements in CIPs/CPSs
- Update author's email address
- Reflect new Copywright section and License header for CPS

### CODE REVIEW

File: CIP-0095/README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ Title: Web-Wallet Bridge - Conway ledger era
...
 Authors:
-  - Ryan Williams <ryan.williams@iohk.io>
+  - Ryan Williams <ryan.williams@intersectmbo.org>
 Implementors:
   - Eternl <https://eternl.io/>
   - GeroWallet <https://gerowallet.io>
   - Lace <https://www.lace.io/>
   - NuFi <https://nu.fi/>
-  - Ryan Williams <ryan.williams@iohk.io>
+  - Ryan Williams <rryan.williams@intersectmbo.org>
```
Comment (Ryun1) [inline CIP-0095/README.md:13]:
```suggestion
  - Ryan Williams <ryan.williams@intersectmbo.org>
```
typo!

**Review by rphair — APPROVED** [review]
🎉

**Review by Crypto2099 — APPROVED** [review]
LGTM

**Comment by rphair** [comment]
oops @Ryun1 I thought that was "my" merge conflict to fix manually 😅 ... anyway will double check the affected CPS-0010 before merging now (i.e. before tomorrow's meeting to keep the slate clear).

**Pull request merged**